azkaban-common/src/test/resources/azkaban/executor/embedded/innerFlow.job

```diff
@@ -0,0 +1,5 @@
+type=javaprocess
+java.class=azkaban.test.executor.SleepJavaJob
```

Comment:
Unfortunately, there does not seem to be a standard way to have this kind of cross-subproject dependency for resources in Gradle. The resources for a test must also be under the same package directory in the resource source set as the test. For example, tests under `test/azkaban/executor` need their resources to be under `resources/azkaban/executor`.

Do you know of a non-hackish way to use resource files from a separate subproject in Gradle? There are some other build systems such as [Bazel](http://bazel.io/docs/build-encyclopedia.html#java_test.resources) that have an easy way to specify this kind of dependency.

Otherwise, I think having a bit of duplication for test files should be fine, especially since these test files are relatively small.